config/jest/config.json

```diff
@@ -6,9 +6,10 @@
   "modulePaths": ["src/components", "src/util"],
   "testPathIgnorePatterns": ["<rootDir>/build", "<rootDir>/dist", "<rootDir>/config", "<rootDir>/src/demos"],
   "moduleNameMapper": {
+    "office-ui-fabric-react/lib/": "office-ui-fabric-react/lib-commonjs/",
```

Comment:
since Fabric 6.x changed the `/lib` directory to be es6 modules, don't we need to change all of our Fabric import statements to import from `/lib-commonjs`? otherwise i think we need to commit to es6 modules in this PR so consumer apps can update their builds?

Reply:
Umm, let me check what the output from `tsc` looks like.

I was able to launch Storybook without changes though.

Reply:
Yup, the output converts our `import`s to `require`s, but of course they still point to `/lib/`. Things work because Webpack understands how to deal with those, no extra config needed.